CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportDialogManager.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2010-2023 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.errorreport;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import javafx.application.Platform;
+
+/**
+ * Maintain a session variable with active dialogs that are shown
+ *
+ * @author OrionsGuardian
+ */
+public class ErrorReportDialogManager {
+
+    private int popupDisplayMode = 2;
+    private final List<Double> activePopupIds = new ArrayList<>();
+    private Date latestPopupDismissDate = null;
+    private final ArrayList<String> popupTypeFilters = new ArrayList<>();
+    private boolean isErrorReportRunning = false;
+    private Timer refreshTimer = null;
+    private Date latestRetrievalDate = null;
+    private Date previousRetrievalDate = null;
+    private Date backupRetrievalDate = null;
+    private Date gracePeriodResumptionDate = null;
+    private ArrayList<ErrorReportEntry> sessionPopupErrors = new ArrayList<>();
+
+    private static ErrorReportDialogManager instance = null;
+    
+    public ErrorReportDialogManager(){
+    }
```

Comment:
you can remove this as this is the default constructor (if one isn't explicitly specified)